app/details/[ticket_id]/ticketDetails.tsx

```diff
@@ -422,15 +430,19 @@ function TicketDetails(props: Props) {
               />
             </ButtonDiv>
             <ButtonDiv>
-              {ticketDetails?.status !== TicketStatus.CLOSED && (
-                <Tag
-                  title='Close'
-                  iconName='close-icon'
-                  isActive={false}
-                  onClick={handleCloseTicket}
-                  isName={false}
-                />
-              )}
+              <Tag
```

Comment:
Create separate tag and show that based on condition

Reply:
Done